tests/testthat/test-add_stat.tbl_continuous.R

```diff
@@ -0,0 +1,57 @@
+test_that("add_stat() for 'tbl_continuous'", {
```

Comment:
Can we add these tests to `test-add_stat()` this name makes it look like an S3 method